GOVERNANCE.md

```diff
@@ -664,11 +664,11 @@ Team:
 * Documentation team members are given permissions to [pymc-examples](https://github.com/pymc-devs/pymc-examples)
   and [resources](https://github.com/pymc-devs/resources) repositories.
 
-In addition, Council members are given admin rights to all repositories within
+In addition, Council members are given administrative rights to all repositories within
 the [pymc-devs](https://github.com/pymc-devs) organization.
 
 #### Discourse
-Similarly to the above section, Discourse permissions are also mapped to the discourse team
+Similarly to the above section, Discourse permissions are also mapped to the community team
 and the two contributor roles.
 
 Role:
```

Comment:
Do we all agree with these roles? 

I don't know how Discourse works and I'm the one who proposed that so we'd have somehting to start the discussion. IMO the process should be as much as possible a "take what we have or want and write it down here" not really write down something here and then change everything else. cc @junpenglao 

What I will do though after a grace period is making sure we are doing what we have written here; and if that were an issue restart the discussion, but it would be very nice to have the discussion "beforehand"

Reply:
I think this is roughly the way it works currently.  But I agree that getting everyone on the same page now is a good idea.  I know that there has been some confusion about what permission the various discourse roles/levels have.

Reply:
We should limit the moderator and admins to community team member if possible.

Reply:
@junpenglao I don't understand the comment, should that be changed? Right now core contributors on the community team and council members should be on the pool of rotating moderator/admin persissions, should it be only community core contributors?